com.unity.render-pipelines.high-definition/Editor/Material/ShaderGraph/Templates/VertexAnimation.template.hlsl

```diff
@@ -57,7 +61,17 @@ $splice(CustomInterpolatorPreVertex)
     $VertexDescriptionInputs.TimeParameters: vertexDescriptionInputs.TimeParameters = timeParameters;
 
     // evaluate vertex graph
+#if defined(HAVE_VFX_MODIFICATION)
+    GraphProperties properties;
```

Comment:
GraphProperties is defined once and zero initialized. The VFX Element just configures the necessary vertex properties which the vertex description function will use. This is very similar to the element pixel properties configuration.